tests/base/converters/test_dataframe_to_mds.py

```diff
@@ -28,12 +31,17 @@ def manual_integration_dir() -> Any:
     """Creates a temporary directory and then deletes it when the calling function is done."""
     if MANUAL_INTEGRATION_TEST:
         os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = 'path/to/gooogle_api_credential.json'
+        os.environ.pop('AWS_ACCESS_KEY_ID', None)
+        os.environ.pop('AWS_SECRET_ACCESS_KEY', None)
+        os.environ.pop('AWS_SECURITY_TOKEN', None)
+        os.environ.pop('AWS_SESSION_TOKEN', None)
+        os.environ['AWS_PROFILE'] = 'temporary'
 
     tmp_dir = mkdtemp()
 
     def _method(cloud_prefix: str = 'gs://') -> Tuple[str, str]:
         mock_local_dir = tmp_dir  # mkdtemp()
-        mock_remote_dir = os.path.join(cloud_prefix, MY_BUCKET, MY_PREFIX)
+        mock_remote_dir = os.path.join(cloud_prefix, MY_BUCKET[cloud_prefix], MY_PREFIX)
```

Comment:
are we going to test all cloud providers or just gcp and s3?

Reply:
I think the plan is to add S3, GCS, and OCI in phase 1 and merge this PR. In phase 2, add a support for other cloud providers as well.

Reply:
dataframeToMDS, merge_index are both validated with s3, oci, gcs as manual integration tests.